python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -751,6 +751,119 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.isis_authentication_key is arista.avd.defined %}
    isis authentication key 7 {{ ethernet_interface.isis_authentication_key | arista.avd.hide_passwords(hide_passwords) }}
 {%     endif %}
+{%     if ethernet_interface.isis_authentication.both.mode is arista.avd.defined
```

Comment:
We need to confirm that should we keep the new and old key in if elif condition or keep  as is it now. I know we have added in description that new key should not be used with old one but yes lets confirm